### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#17] [Announcements]
# about

做了一个播客叫做打点酱油，平常写 Python, 对NLP、计算广告、大模型感兴趣，尝试做一些有意义的事情

https://yuanchaofa.com/about.html

**Comment by JackBo04** [comment]
很有收获，谢谢up，加油哇！祝up一切顺利（我来做第一个留言的）

**Reply by bbruceyuan** [reply]
感激感激，有帮助就好，哈哈哈

> 其实视频还是有一些错误，而且很难纠正更改，不知道有啥好办法可以纠错

**Comment by yaozile123** [comment]
up加油 我觉得你就是b站的andrej karpathy

**Reply by bbruceyuan** [reply]
hhh，这可完全不敢当😱但是确实很多内容都是学习 ak 的方式，边写边讲知识点

**Comment by ignitemylife** [comment]
发哥牛牪犇

**Reply by bbruceyuan** [reply]
🐂

**Comment by heyu58** [comment]
谢谢up！我还是处于迷茫状态的学生，找self-attention资料时候找到你的。真诚的表达和用心的随笔真的很打动人。祝你生活愉快，坚持创作产出。

**Reply by bbruceyuan** [reply]
加油，一起加油～

**Comment by yzq-zz** [comment]
xhs刷到大佬字节连续两年半离职了，刚好搜了你的bytetech，看到你的主页
我是做后端的，从你关于agent的文章学到很多，希望可以继续更新！

**Reply by bbruceyuan** [reply]
以后还会继续更新的，非常**感谢你的鼓励**，其实我 blog 里面是 bytetech 的超集（就是有很多乱七八糟的个人东西）；

内部 bytetech 同步太麻烦了，所以一般其实还是blog为主，除非不能对外分享的（不过这种非兄弟团队也看不到😂）

以后也还在做这一块，肯定还是需要持续学习的
